table/table_test.cpp

```diff
@@ -155,7 +157,17 @@ int main() {
     assert(table.filter(columns1, comparisons1)[0].entries(0).num() == 1);
     
     table.edit_rows(columns3, comparisons3, columns3, comparisons4);
-    assert(table.filter(columns1, comparisons1).size() == 3);
+    assert(table.filter(columns3, comparisons4).size() == 4);
     
     cout << "Passed Update Tests!" << endl;
+    /* Passed Update Tests */
+
+    /* Deletion Tests */
+    cout << "Starting Deletion Tests..." << endl;
+    
+    table.remove_rows(columns3, comparisons3);
```

Comment:
Please add more tests to cover all code paths introduced by this PR. It look like only basic cases are tested in this PR, but all potential codepaths should be covered.